mints/05.json

```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/wxdai:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeicsbm2itxmmifkkyroozp6dh45aatrqabv6bcpkm7p2nin7j2trvm","image":"ipfs://bafybeicpuzx75bni7cjvhdnzp3id7ay5fzgmqvor37nxgjxiqwhwwjaayu","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
"The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory